apps/codeforafrica/src/components/Articles/useArticles.js

```diff
@@ -4,11 +4,13 @@ import useFilterQuery from "@/codeforafrica/components/useFilterQuery";
 
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
-function useArticles(params, primaryTag) {
+function useArticles(params, { primaryTag, featured }) {
   const queryParams = useFilterQuery(params);
   const query = queryParams
-    ? `${queryParams}&primaryTag=${primaryTag}`
-    : `?primaryTag=${primaryTag}`;
+    ? `${queryParams}&primaryTag=${primaryTag}${
+        featured ? `&featured=${featured}` : ""
+      }`
+    : `?primaryTag=${primaryTag}${featured ? `&featured=${featured}` : ""}`;
```

Comment:
This can be improved e.g.
```js
  const filterQuery = queryParams ? `?${queryParams}&` : '?';
  const primaryTagQuery = `primaryTag=${primaryTag}`;
  const featuredQuery = featured ? `&featured=${featured}&` : '';
  const query = `${filterQuery}${primaryTagQuery}${featuredQuery}`
```